### ISSUE TITLE
de-RSE25 workshop

### ISSUE THREAD

**Issue by CaptainSifff** [the-teachingRSE-project/institutionalized_education#13]
I've submitted a workshop. 
Title: the teachingRSE project working meeting
Abstract: In this working group meeting we plan to work on our forthcoming publication of detailing how to create structures for the future training of young and practising RSEs

So we could definitely use a more catchy title and abstract.
I've put it here in the repo, since the focus is on this topic.

**Comment by CaptainSifff** [comment]
Length is also a discussion we should have. Currently I've requested one hour. But I also think 3 hours would be nicer,
to really form some community again. But this will more likely force me to distribute the workshop to the boundaries of the conference.

**Comment by jcohen02** [comment]
Thanks for submitting the workshop @CaptainSifff. Is there an intermediate option between 1 hour and 3 hours? I think having something integrated into the main schedule (i.e. not as a side meeting, since I wonder if that may significantly reduce the number of attendees?) would be preferable.

From experience of running recent conference workshop sessions and BoFs, I'm reminded just how little time 1 hour actually is if we want to have an interactive session! I think we could do something that would fit with the aims of the group in 2 hours, or even 90 minutes. I have a feeling 2 hours might be the ideal kind of length since it's enough time to do something substantial but not too long that we lose engagement from people who are not core members of the teachingRSE team.

If part of the schedule is organised to have 1-hour blocks for workshops, might there be an opportunity to do a double workshop covering two sessions?

**Comment by CaptainSifff** [comment]
My current planning is to put this on the 25.02 in the morning. We could e.g. do 10:30 - 12:00. sorry, there have been many good workshops!

**Comment by MakisH** [comment]
> My current planning is to put this on the 25.02 in the morning. We could e.g. do 10:30 - 12:00. sorry, there have been many good workshops!

Why the "sorry"? Is this creating any conflicts?
It's in the middle of the conference, so sounds ideal to me.

**Comment by CaptainSifff** [comment]
25th is Tuesday, right before the conference....

**Comment by MakisH** [comment]
Not according to https://events.hifis.net/event/1741/ (currently 25.02 9:00AM - 27.02 1:00PM).

**Comment by jcohen02** [comment]
I think I recall @CaptainSifff saying that the conference is supposed to start at lunch time on the Tuesday?
(I should add that I unfortunately won't be able to make it to this session if it is on the Tuesday morning since I think I will not arrive until around 12 or 1, although don't reschedule on my account, I know shceduling these things is very challenging). One thought, if the conference finishes at lunch on Thursday, maybe just after lunch on Thursday is also an option?

**Comment by CaptainSifff** [comment]
Yes, thursday after Lunch would also bei an Option. This would be ~13 until however long we want to make it. There is one round of Workshops after the conference. And the se25 is still ongoing.

**Comment by CaptainSifff** [comment]
We do have various sessions,  by members of our group, that I can happily include you in the main session.
So if you're interested in joining, please tell me.

**Comment by CaptainSifff** [comment]
### (Research) Software Engineering (Research): Creating knowledge and busting myths at the intersection of software engineering and research. #7

Authors: Stephan Druskat, Florian Goth, Anna-Lena Lamprecht


Software Engineering Researchers (SERs) and Research Software Engineers (RSEs) can potentially benefit from each other:
SERs can provide RSEs with state-of-the-art research knowledge, methods and tools from software engineering that can help create better software for better research.
RSEs can help SERs understand the specific challenges they face in research software engineering, and thus provide interesting new research questions.
This can create a virtuous circle of mutual benefit through collaboration.
However, known and unknown gaps in knowledge about what the respective other community does, and how the practices of software engineering in, e.g., industry and in research differ may have led to preconceptions and myths that obstruct the pathway to fruitful and reciprocal collaboration.
In this session, we create an opportunity to bust some of these myths - and help build understanding, transparency and trust - by engaging members of both communities in discussion, in a fishbowl format.
The session is split into two moderated sections, each of which discusses myths, preconceptions and questions about one of the two communities and their practice, challenges and outputs.
Preconceptions and questions are fielded anonymously before and during the session, and participants can engage personally by joining the discussion fishbowl if and when they want.
At the end of this session, participants have ideally increased knowledge about, and understanding of, another community that operates at the intersection of software and research, and potentially also their own community.

**Comment by CaptainSifff** [comment]
### Software Engineering for and with Researchers: What is required? #15

Authors: Jeremy Cohen, Florian Goth, Guido Juckeland, Jan Philipp Thiele


With the role of the Research Software Engineer in the academic landscape now better defined, it is time to ask the question to a broader audience of how can we adapt traditional Software Engineering practices to most effectively fit the needs of the research community. What differentiates researchers who write code from RSEs? How do their aims, drivers and motivations differ and how does this affect the application of technical skills? Can we, indeed do we need to, change the way traditional software skills are applied in order to more effectively support the RSE domain? Research is becoming increasingly reliant on digital skills and infrastructure but we won’t be able to capitalise on their capabilities if we don’t ensure that they are used correctly. The teachingRSE project together with EVERSE aims to have a discussion among a diverse set of participants on how to adapt SE practices for domain researchers and software professionals working within the research community.

**Comment by CaptainSifff** [comment]
### The End of RSEng? Challenges and Risks for RSEng #158

Authors: Maria Guadalupe Barrios Sazo, Gerasimos Chourdakis, Jeremy Cohen, Florian Goth, Magnus Karl Moritz Hagdorn, Toby Hodges, Michele Martone, Jan Philipp Thiele


It is certainly too early to herald the end of RSEng already, but
the future is in constant flux and we should discuss how RSEng will change in response to the upcoming challenges in an open discussion.
Some questions to get the discussion started:

How will RSE change in the face of the ongoing digitalization of society? If people are bette rprepared in school?
How will AI impact our work
What will our topics be after version control doesn’t need to be taught anymore? Digitalisation will also feature more heavily in the domain curricula.
What is in this future for RSEs?
This BoF aims to bring together people interested in reflecting on the challenges and opportunities for RSE.

**Comment by CaptainSifff** [comment]
all three are intended as more interactive 45minute sessions, that should cater to deRSE and SE.

**Comment by jngrad** [comment]
I would be interested in joining #&ZeroWidthSpace;15.

**Comment by CaptainSifff** [comment]
> I would be interested in joining #​15.

You're in! Note that you are now at least listed in 4 contribs at the conference.

**Comment by CaptainSifff** [comment]
Here's our contribution: 
https://events.hifis.net/event/1741/contributions/13967/

Please check if you're mentioned and you plan to participate in the workshop.

**Comment by CaptainSifff** [comment]
I was just reminded that there was the idea of a group photo: https://github.com/the-teachingRSE-project/general/issues/9 
Maybe at de-RSE25?

**Comment by jcohen02** [comment]
Many thanks for highlighting this Florian, it looks good, but you can probably guess what my comment will be. 🙂

Happy to go with whatever you prefer but I'd suggest some very minor edits to the text on the webpage, for example:

"_the teachingRSE project is community of interest formed around the idea of finding structures for the most effective education of new and developing RSEs in an academic landscape
In this working group meeting we plan to work on our forthcoming publication on detailing how to create structures for the future training of both new and practising RSEs._"

**Comment by BeastyBlacksmith** [comment]
I don't see myself mentioned there, but I do intend to participate.

**Comment by CaptainSifff** [comment]
OK, both fixed.

**Comment by MakisH** [comment]
> I don't see myself mentioned there, but I do intend to participate.

Same for me, and I guess also for @mhagdorn (just looking at the pillar issues).

**Comment by mhagdorn** [comment]
> > I don't see myself mentioned there, but I do intend to participate.
> 
> Same for me, and I guess also for [@mhagdorn](https://github.com/mhagdorn) (just looking at the pillar issues).

yes, me too. happy to just come along

**Comment by CaptainSifff** [comment]
both fixed.

**Comment by jpthiele** [comment]
Update on the IEEE branding.
I don't see an obvious trigger to turn off most of the branding, especially the blue and yellow parts of the PDF.
we could remove all magazine names and such but I'd still be unsure if we can easily publish it that way.

**Comment by CaptainSifff** [comment]
hmmm... Disassembling the pdf in inkscape or word....? It doesn't need to be a reproducible process.

**Comment by jngrad** [comment]
> hmmm... Disassembling the pdf in inkscape or word....? It doesn't need to be a reproducible process.

That could potentially degrade accessibility of the PDF file. Not that PDF is terribly accessible to begin with, but depending on the accessibility options enabled in the LaTeX toolchain, extra information is stored in the alternative text layer and metadata layer to help screen readers (e.g. text-to-speech assistants and Braille displays): math formula can be rendered as UTF-8 code points in the visual layer and as their original LaTeX code in the text layer, page breaks in the middle of a paragraph can be tagged to jump to the correct line (and thus skip page headers and footers in that paragraph), and words split across a line can remain whole in the text layer. If there is a way to provide the original LaTeX file together with the reassembled PDF file, then we should do so, so that individuals with accessibility issues can read the LaTeX file directly.

**Comment by CaptainSifff** [comment]
Mentioning that work continues in #15 #16 #17 #18

**Comment by MakisH** [comment]
I understand we can close this, we have related issues/PRs.
